test/local/account_routes.js

```diff
@@ -90,17 +91,19 @@ test('/recovery_email/status', function (t) {
   })
   var route = getRoute(accountRoutes, '/recovery_email/status')
 
-  test('sign-in confirmation disabled', function (t) {
+  t.test('sign-in confirmation disabled', function (t) {
+    t.plan(2)
     config.signinConfirmation.enabled = false
 
-    test('invalid email', function (t) {
+    t.test('invalid email', function (t) {
```

Comment:
@philbooth Thoughts? Seems reasonable, but it would also require a good refactor from what was done in https://github.com/mozilla/fxa-auth-server/pull/1289. Either way, I think having nested tests is the way to go.